hypervisor/arch/x86/Kconfig

```diff
@@ -221,12 +221,24 @@ config RAM_START
 	  bootloader may not place the hypervisor at this specific address.
 
 config RAM_SIZE
-	hex "Size of the RAM region assigned to the hypervisor"
-	default 0x02000000
+	hex "Size of the RAM region used by the hypervisor"
+	default 0x04000000
 	help
-	  A 64-bit integer indicating the size of RAM assigned to the
-	  hypervisor. It is ensured at link time that the footprint of the
-	  hypervisor does not exceed this size.
+	  A 64-bit integer indicating the size of RAM used by the hypervisor.
+	  It is ensured at link time that the footprint of the hypervisor
+	  does not exceed this size.
+
+config PLATFORM_RAM_SIZE
+	hex "Size of the RAM"
+	default 0x200000000
+	help
+	  A 64-bit integer indicating the size of the RAM (not included the MMIO).
+
+config PLATFORM_MMIO_SIZE
+	hex "Size of the MMIO"
```

Comment:
PLATFORM_LO_MMIO_SIZE

Reply:
Size of the low MMIO address space.

But I actually this one should be fixed, likely 1G.  Do we need to make it configurable?